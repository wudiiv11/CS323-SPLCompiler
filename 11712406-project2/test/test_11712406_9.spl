struct string {
  int size;
  int value[256];
};

float inc(int n) {
  return n + 1;
}

int test_2_r02()
{
  struct string str;
  str.size = 5;
  return inc(str.size) + (0-1);
}