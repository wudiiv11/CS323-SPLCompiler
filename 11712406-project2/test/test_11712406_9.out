Error type 8 at Line 7: incompatible return type
Error type 8 at Line 14: incompatible return type